tests/sles4sap/publiccloud/add_server_to_hosts.pm

```diff
@@ -21,8 +21,9 @@ sub run {
         next if ($instance->{'instance_id'} !~ m/vmhana/);
         record_info("$instance");
 
+        my $repo_host = get_var('REPO_MIRROR_HOST', 'download.suse.de');
         my $ibsm_ip = get_required_var('IBSM_IP');
-        $instance->run_ssh_command(cmd => "echo \"$ibsm_ip download.suse.de\" | sudo tee -a /etc/hosts", username => 'cloudadmin');
+        $instance->run_ssh_command(cmd => "echo \"$ibsm_ip $repo_host\" | sudo tee -a /etc/hosts", username => 'cloudadmin');
```

Comment:
We are no more using this test module.
We are doing the same via https://github.com/SUSE/qe-sap-deployment/blob/main/ansible/playbooks/ibsm.yaml and I do no think that `REPO_MIRROR_HOST` is needed there, as we are already using an aliased name